src/superlifter/core.cljc

```diff
@@ -178,9 +178,18 @@
                      %))
   context)
 
+(defn- stop-bucket! [bucket]
+  (doseq [{:keys [stop-fn]} (vals (:triggers bucket))
+          :when stop-fn]
+    (stop-fn)))
+
 (defn add-bucket! [context id opts]
   (swap! (:buckets context)
-         #(assoc % id (start-bucket! context id opts)))
+         (fn [buckets]
+           (when-let [existing-bucket (get buckets id)]
+             (stop-bucket! existing-bucket)
```

Comment:
Do you think we should also flush buckets in this case? Otherwise anything enqueued will be orphaned.

Reply:
Yeah, probably a good idea.

Reply:
Decided to do it in the body of `stop-bucket!` since I can't really think of a case where we'd want to stop a bucket without flushing it. lmk if you want to do something else.